docs/howto/getting_started_markdown.md

```diff
@@ -146,6 +146,180 @@ development. For more information on Configuration Panel GUI
 development, go to [Alteryx UI
 SDK](https://help.alteryx.com/developer-help/ayx-ui-sdk).
 
+## Platform SDK Quickstart Guide
+
+#### Python SDK v2
+
+The steps below illustrate how to use the latest developer tools to
+create a custom plugin or tool for use in Alteryx. For more information
+about new SDKs and their usage within Alteryx, please visit [Platform
+SDK](../references/platform-sdk.md).
+
+### Requirements and Prerequisites
+
+Before you get started, make sure that these items are installed on your
+machine:
+
+-   Microsoft Windows 7 or Later (64-bit)
+-   Alteryx Designer Version 2021.2\*
```

Comment:
Why the asterisk?

Reply:
here it indicates the note below the list with the asterisk is related to this line. Bit confusing imo but this information matches our other documentation